### ISSUE TITLE
How to validate while training

### ISSUE THREAD

**Issue by Davidwontwait** [Oliiveralien/MDMS#3]
Hi! I admire your work and I would like to run it on my datasets. But I notice that your released code in DATASETS part set patches for validate "return torch.stack(outputs, dim=0), img_id,ii,jj,size". If I want to validate the whole image, what should I do ? THX!

**Comment by skkkyup** [comment]
Sorry for just seeing the message. In the lowlight.py file within the DATASETS, if it is used for validation and “self.parse_patches=false”, then the length and width of the image are resized to be multiples of 8 to facilitate subsequent patching. Therefore, the code is capable of processing the entire image, not just generating patches. As for “torch.stack(outputs, dim=0), img_id, ii, jj, osize”, these are actually used during training. The parameters correspond to the image, image ID, randomly patched position, and the size of the patch, respectively. Hope this helps.
